### PR TITLE
YLP-1058 password strength meter

### PR DESCRIPTION
**Pull request by fatb38**

### CODE REVIEW

**Comment by fatb38** [comment]
Some visuals
Screenshot from 2021-11-08 10-45-43
Screenshot from 2021-11-08 10-45-55

File: packages/yourloops/pages/profile/auth-form.tsx [hunk 1]
```diff
@@ -80,17 +84,24 @@ function AuthenticationForm(props: AuthenticationFormProps): JSX.Element {
...
         autoComplete="new-password"
         variant="standard"
         label={t("new-password")}
         value={password}
-        error={errors.password}
-        helperText={t("password-too-weak")}
-        setState={setPassword}
+        error={errors.password && password.length > 0}
+        checkStrength
+        helperText={
```
Comment (yvon-dblg) [inline packages/yourloops/pages/profile/auth-form.tsx:97]: Need to set FormHelperTextProps too

**Review by yvon-dblg — CHANGES_REQUESTED** [review]

**Comment by yvon-dblg** [comment]
Missing changelog

File: packages/yourloops/pages/profile/auth-form.tsx (hunk 1 above)
Comment (fatb38) [reply packages/yourloops/pages/profile/auth-form.tsx:97]: after refactoring the helperText condition into paswword component, error disappear

**Pull request merged**